src/wallet/crypter.h

```diff
@@ -16,20 +16,22 @@
 
 class uint256;
 
-const unsigned int WALLET_CRYPTO_KEY_SIZE = 32;
+const unsigned int WALLET_CRYPTO_KEY_SIZE = crypto_secretbox_KEYBYTES;
 const unsigned int WALLET_CRYPTO_SALT_SIZE = crypto_pwhash_SALTBYTES;
 
 /**
  * Private key encryption is done based on a CMasterKey,
  * which holds a salt and random encryption key.
  * 
- * CMasterKeys are encrypted using AES-256-CBC using a key
- * derived using derivation method nDerivationMethod
- * (0 == Argon2i()) and derivation iterations nDeriveIterations.
+ * CMasterKeys are encrypted using XSalsa20/Poly1305 (libsodium's
+ * crypto_secretbox) using a key derived using derivation method
```

Comment:
I'd prefer [AEAD_CHACHA20_POLY1305](https://tools.ietf.org/html/rfc7539), the same as we use for the note encryption.